index.html

```diff
@@ -106,71 +106,109 @@
           or other <a>track</a> types that support multiple selections)
           <a>track</a>s from the media element.  If the media element does not
           have a selected or enabled <a>track</a>s of a given type, then no
-          <code>MediaStreamTrack</code> is present in the captured stream.
+          <code>MediaStreamTrack</code> of that type is present in the captured
+          stream.
         </p>
         <p>
           A <code>&lt;video&gt;</code> element can therefore capture a video
           <code>MediaStreamTrack</code> and any number of audio
-          <code>MediaStreamTrack</code>s. An <code>&lt;audio&gt;</code> element
+          <code>MediaStreamTrack</code>s.  An <code>&lt;audio&gt;</code> element
           can capture any number of audio <code>MediaStreamTrack</code>s.  In
           both cases, the set of captured <code>MediaStreamTrack</code>s could
           be empty.
         </p>
         <p>
-          <code>captureStream</code> produces a <code>MediaStream</code> that
-          captures any media that is currently playing on the element.  Changes
-          in the media element source do not cause the stream to terminate,
-          though the set of <code>MediaStreamTrack</code>s might change over
-          time.  If the source stream for the media element ends, or the a
-          different source is selected, the <code>MediaStream</code> captures
-          the state of the media element.  This means that there could be
-          periods where the captured stream has no active media content.
+          Changes in the media element source do not cause the stream to
+          terminate, though the set of <code>MediaStreamTrack</code>s might
+          change over time.  If the source stream for the media element ends, or
```

Comment:
"source stream for the media element" - a little confusing what kind of stream this is talking about. Can we just say "source" instead?